Jenkinsfile

```diff
@@ -136,6 +137,11 @@ EOF
 										  echo $ret >.cypress_status
 										else
 										  FAILED_TESTS="$( awk '/Spec/{f=1}f' cypress.log |grep --only-matching '✖ .*.spec.js' |awk '{ print "test/cypress/e2e/"$2 }' |xargs| tr -s ' ' ',' )"
+										  cd $WORKSPACE/$BRANCH_NAME
++										  make coldstart
++										  export CYPRESS_coreUrl=http://127.0.0.1:$( docker-compose port lisk 4000 |cut -d ":" -f 2 )
++										  sleep 10
++										  cd -
```

Comment:
@Efefefef This is great. I was thinking about something like this since some time :-) 

But it looks like copy-paste from a diff (with `+` at the beginning of line)

Reply:
my bad